lib/options.ts

```diff
@@ -164,33 +164,34 @@ interface DeprecatedOptions {
   esprima?: any;
 }
 
-var defaults: Options = {
-  parser: require("../parsers/esprima"),
-  tabWidth: 4,
-  useTabs: false,
-  reuseWhitespace: true,
-  lineTerminator: require("os").EOL || "\n",
-  wrapColumn: 74, // Aspirational for now.
-  sourceFileName: null,
-  sourceMapName: null,
-  sourceRoot: null,
-  inputSourceMap: null,
-  range: false,
-  tolerant: true,
-  quote: null,
-  trailingComma: false,
-  arrayBracketSpacing: false,
-  objectCurlySpacing: true,
-  arrowParensAlways: false,
-  flowObjectCommas: true,
-  tokens: true
-}, hasOwn = defaults.hasOwnProperty;
+const defaults: Options = {
+        parser: require("../parsers/esprima"),
+        tabWidth: 4,
+        useTabs: false,
+        reuseWhitespace: true,
+        lineTerminator: require("os").EOL || "\n",
+        wrapColumn: 74, // Aspirational for now.
+        sourceFileName: null,
+        sourceMapName: null,
+        sourceRoot: null,
+        inputSourceMap: null,
+        range: false,
+        tolerant: true,
+        quote: null,
+        trailingComma: false,
+        arrayBracketSpacing: false,
+        objectCurlySpacing: true,
+        arrowParensAlways: false,
+        flowObjectCommas: true,
+        tokens: true
+      },
+      hasOwn = defaults.hasOwnProperty;
```

Comment:
I think I see what happened here (lining up the variables vertically, then indenting the object literal by two spaces relative to that), but I wonder if this is really the indentation we want?

Reply:
Probably not. I'll split into two declarations and that should fix it.